crates/bevy_core/src/time/fixed_timestep.rs

```diff
@@ -282,16 +307,21 @@ mod test {
     fn advance_time(world: &mut World, instance: Instant, seconds: f32) {
         world
             .get_resource_mut::<Time>()
-            .unwrap()
+            .expect("Could not find `Time` in the `World`.")
             .update_with_instant(instance.add(Duration::from_secs_f32(seconds)));
     }
 
     fn get_accumulator_deciseconds(world: &World) -> f64 {
         world
             .get_resource::<FixedTimesteps>()
-            .unwrap()
+            .expect("Could not find `FixedTimesteps` in the 'World'.")
             .get(LABEL)
-            .unwrap()
+            .unwrap_or_else(|| {
+                panic!(
+                    "Could not find a resource of the appropriate label LABEL={}",
+                    LABEL
+                )
+            })
```

Comment:
We can use Rust 2021 syntax now

```suggestion
            .unwrap_or_else(|| {
                panic!("Could not find FixedTimestep with label: {LABEL}")
            })
```